config/settings.py

```diff
@@ -132,7 +132,7 @@
 DATABASES = {
     'default': {
         "ENGINE": "django.db.backends.mysql",
-        "NAME": "cnl",
+        "NAME": "purchaseupdation",
```

Comment:
what is this purchaseupdation? It should be cnl right?

Reply:
Sorry Thulasi I gave local database name there.(Now changed to cnl)